### PR TITLE
Remove post-completion result collection, add autostart option

### PR DESCRIPTION
**Pull request by pmoeller91-yottaa**
feat!: adds ability to autostart QUnit tests

BREAKING CHANGE: To avoid losing test context, removes the ability to collect test results if the tests are already completed in the browser when the service is inserted. Instead, a recommendation is made to disable test auto-start in QUnit and allow the service to start the tests itself. If this is infeasible, adding a manual delay in the tests will also serve to give the service time to enter the browser context prior to test completion.

# Summary

Prior, when test results were collected post-completion, this resulted in a different format for the result which lost module names, and some information missing from the results. My answer to this is to encourage delayed test start by disabling `autostart` in QUnit and instead allowing the `wdio-qunit-service` to handle automatically starting the test. This will guarantee consistent results from QUnit.

In the case where tests must be started without using the service a couple issues may result.

Issue: Tests start and finish before the test service can setup the callback

Possible fixes:
 - Modify tests so they can be started by the QUnit WDIO service
 - Add a slight manual delay to the beginning of the test run to ensure the service can attach the callback prior to test run completion.
   - Ex: `await new Promise((resolve) => { setTimeout(resolve, 250); });` in a `before` hook.

Issue: Tests are started prior to being ready by the test service

Possible fixes:
 - Extend the `autostartDelay` value to allow additional time for the tests to automatically be started in the browser by your mechanism of choice. The service will only attempt to autostart if the tests have not started at the end of this delay.

```js
// in wdio.conf.js
export const config = {
  // ..
  services: [['qunit', { autostartDelay: 1000 }],
  // ..
};
```

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -22,14 +23,28 @@ jobs:
...
-      - run: npm run test:features
-      - run: npm run test:qunit
-      - run: npm run test:no:specs
-      - run: npm run test:cjs
-      - run: npm run lint:ci
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+      - name: Build
+        run: npm run build
+      - name: "Test: ESM"
```
Comment (bburke4) [inline .github/workflows/test.yml:33]: the names are optional... it just uses the command in the action output/logs, but doesnt matter

File: package.json [hunk 2]
```diff
@@ -67,9 +69,12 @@
...
     "type": "git",
-    "url": "git+https://github.com/mauriciolauffer/wdio-qunit-service.git"
+    "url": "https://github.com/yottaa/wdio-qunit-service.git"
+  },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com"
   },
   "bugs": {
-    "url": "https://github.com/mauriciolauffer/wdio-qunit-service/issues"
+    "url": "https://github.com/pmoeller91-yottaa/wdio-qunit-service/issues"
```
Comment (bburke4) [inline package.json]: should change to yottaa? or delete i guess

File: README.md [hunk 3]
```diff
@@ -1,8 +1,11 @@
-# wdio-qunit-service
+# @yottaa/wdio-qunit-service
 
-[![npm](https://img.shields.io/npm/v/wdio-qunit-service)](https://www.npmjs.com/package/wdio-qunit-service) [![test](https://github.com/mauriciolauffer/wdio-qunit-service/actions/workflows/test.yml/badge.svg)](https://github.com/mauriciolauffer/wdio-qunit-service/actions/workflows/test.yml)
+[![NPM Version](https://img.shields.io/npm/v/@yottaa/wdio-qunit-service)](https://www.npmjs.com/package/@yottaa/wdio-qunit-service)
+[![test](https://github.com/pmoeller91-yottaa/wdio-qunit-service/actions/workflows/test.yml/badge.svg)](https://github.com/pmoeller91-yottaa/wdio-qunit-service/actions/workflows/test.yml)
```
Comment (bburke4) [inline README.md]: change org to yottaa?

**Review by bburke4 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (pmoeller91-yottaa) [reply package.json]: Fixed!

File: README.md (hunk 3 above)
Comment (pmoeller91-yottaa) [reply README.md]: Good catch, fixed this and a couple other things I missed. See latest commit. Had to force push to fix some commit messages.

File: .github/workflows/test.yml (hunk 1 above)
Comment (pmoeller91-yottaa) [reply .github/workflows/test.yml:33]: Yeah just makes it look a little nicer, if you have objections let me know.

**Pull request merged**